extensions/ticktick/src/components/taskItem.tsx

```diff
@@ -103,6 +124,25 @@ const TaskItem: React.FC<{
                 icon={{ source: Icon.Dot, tintColor: checkboxColor }}
               />
               <List.Item.Detail.Metadata.Separator />
+              {startDate ? (
+                <>
+                  <List.Item.Detail.Metadata.Label
+                    title="Start Date"
+                    text={formatPrettyDateTime(startDate, timeZone, isFloating, isAllDay)}
+                  />
+                  <List.Item.Detail.Metadata.Separator />
+                </>
+              ) : null}
+              {dueDate ? (
+                <>
+                  <List.Item.Detail.Metadata.Label
+                    title="Due Date"
+                    text={formatPrettyDateTime(dueDate, timeZone, isFloating, isAllDay)}
+                  />
+                  <List.Item.Detail.Metadata.Separator />
+                </>
+              ) : null}
```

Comment:
there some date mode logic not cover
1. when `startDate` is undefined, render nothing
2. when only have `startDate` or `startDate` is same as the `dueDate`, render one label( title="Date" text={ formatPrettyDateTime(startDate, timeZone, isFloating, isAllDay)})
3. else render startDate and dueDate

![CleanShot 2024-01-05 at 10  05 44@2x](https://github.com/raycast/extensions/assets/53415256/04cf5eab-9c0b-498c-93c2-384153d68188)
![CleanShot 2024-01-05 at 10  05 50@2x](https://github.com/raycast/extensions/assets/53415256/830d6d3c-6bdf-4ec2-89d7-5efc1c26073a)